### PR TITLE
[WIP] Add custom repl commands

### LINKED ISSUE #281: [Feature request] Custom REPL commands with editor shortcut
In Cursive, I found that "REPL Commands" is particularly useful feature. I can set a form like `(refresh) (test)` and ns `'user` and bind a editor shortcut like `CMD + R` to execute it. It is very convenient , when combining with Component/Mount/Integrant.

Is it feasible to be implemented in VS Code?

### PR DESCRIPTION
**Pull request by PEZ**
Fixes #281.

Introduces the following change(s):
- Adds a command: **Run Custom REPL Command**
- Adds a settings configuration. (See #281, for documentation)
- Changes the way the function `sendTextToREPLWindow` works so that it does `require` and `in-ns` of the `ns`, if that is provided.

I have:

- [x] Read [How to Contribute](https://github.com/BetterThanTomorrow/calva/wiki/How-to-Contribute#before-sending-pull-requests).
- [x] Made sure I am directing this pull request at the `dev` branch. (Or have specific reasons to target some other branch.)
- [x] Made sure I am changed the default PR base branch, so that it is not `master`. (Sorry for the nagging.)
- [x] Referenced the issue I am fixing/addressing _in a commit message for the pull request_.
     - [x] If I am fixing the issue, I have used [GitHub's fixes/closes syntax](https://help.github.com/en/articles/closing-issues-using-keywords)
- [x] Updated the `[Unreleased]` entry in `CHANGELOG.md`, linking the issue(s) that the PR is addressing.

Ping: @kstehn, @cfehse

### CODE REVIEW

**Comment by PEZ** [comment]
This is good to go, AFAICT, @kstehn , @cfehse

File: src/repl-window.ts [hunk 1]
```diff
@@ -198,17 +199,23 @@ class REPLWindow {
...
+}
 
 function getImageUrl(name: string) {
     let imagepath = "";
     if (!name)
-         imagepath = path.join(ctx.extensionPath, "assets/images/empty.svg");
+        imagepath = path.join(ctx.extensionPath, "assets/images/empty.svg");
     else
-         imagepath = path.join(ctx.extensionPath, "assets/images/", name);
+        imagepath = path.join(ctx.extensionPath, "assets/images/", name);
```
Comment (kstehn) [inline src/repl-window.ts]: i think we should add here `{ }` 
it would be more consistent with the rest of the code

Comment (PEZ) [reply src/repl-window.ts]: Agree.

This PR doesn't care about imageUrls at all, but Kaizen and all that. Update coming.

 We should have that convention documented somewhere, btw.

Comment (kstehn) [reply src/repl-window.ts]: Yes, we should add a "Convention" Section to the "How to Contribute" :)

**Pull request merged**